pkg/manifests/vllm/deployment.yaml

```diff
@@ -1,4 +1,13 @@
 apiVersion: v1
+kind: Secret
+metadata:
+  name: hf-token
+  labels:
+    app: vllm
+stringData:
+  token: $HF_TOKEN
```

Comment:
e2e will replace this variable and the POC instructions have been updated accordingly.

Reply:
I think we should have this Secret resource in a separate yaml so that the guide doesn't require the user to clone the repo to test it out.